main.py

```diff
@@ -2,6 +2,40 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
The genre 'Dramma' is likely a typo and is corrected later in the code to 'Drama'. Ensure this is intentional for demonstration purposes.